tests/jest/application-status-overlay/ApplicationStatusOverlay.test.jsx

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import uuidv4 from 'uuid/v4';
+
+import ApplicationStatusOverlay from '../../../src/application-status-overlay/ApplicationStatusOverlay';
+import ApplicationStatusOverlayContext from '../../../src/application-status-overlay/ApplicationStatusOverlayContext';
+
+jest.mock('uuid/v4');
+
+describe('ApplicationStatusOverlay', () => {
+  let reactUseContext;
+  let statusOverlayContextValue;
+
+  beforeAll(() => {
+    uuidv4.mockReturnValue('test-id');
+
+    /**
+     * Until Enzyme is updated with full support for hooks, we need to
+     * mock out the useContext implementation.
+     */
+    reactUseContext = React.useContext;
+    React.useContext = (contextValue) => {
+      if (contextValue === ApplicationStatusOverlayContext) {
+        return statusOverlayContextValue;
+      }
+      return reactUseContext(contextValue);
+    };
+  });
+
+  afterAll(() => {
+    React.useContext = reactUseContext;
+  });
+
+  beforeEach(() => {
+    statusOverlayContextValue = {
+      show: jest.fn(),
+      hide: jest.fn(),
+    };
+  });
+
+  it('should render status view without any data', () => {
+    const wrapper = mount(
+      <ApplicationStatusOverlay />,
+    );
+
+    expect(wrapper).toMatchSnapshot();
+
+    expect(statusOverlayContextValue.show.mock.calls.length).toBe(1);
+    expect(statusOverlayContextValue.show.mock.calls[0][0]).toBe('test-id');
+    expect(statusOverlayContextValue.hide.mock.calls.length).toBe(0);
+
+    wrapper.unmount();
+
+    expect(statusOverlayContextValue.show.mock.calls.length).toBe(1);
+    expect(statusOverlayContextValue.hide.mock.calls.length).toBe(1);
+    expect(statusOverlayContextValue.hide.mock.calls[0][0]).toBe('test-id');
+  });
+
+  it('should render status view with the specified data', () => {
+    const StatusViewButtons = [
+      {
+        text: 'Action 1',
+        key: 1,
+        size: 'medium',
```

Comment:
`size` is no longer a valid button prop, also remove on line 67

Reply:
Removed here : https://github.com/cerner/terra-application/pull/47/commits/62b43a41763ab55e834f07ce82e374ac33e0b409